acceptance/features/edit_single_question_autocomplete_page_spec.rb

```diff
@@ -77,7 +77,7 @@ def and_I_go_to_the_page_that_I_edit(preview_form)
   def then_I_should_see_my_changes_in_the_form(preview_form, change)
     within_window(preview_form) do
       page.find('.autocomplete__input').click
-      sleep(1) # allow time for page to load
+      sleep(5) # allow time for page to load
```

Comment:
Does this alleviate anything? Just thinking if 1 didn't work I wouldn't _expect_ 5 to, but if it (even anecdotally) does then cool

Reply:
Hard to say... the test had failed repeatedly until I made this change...  but that could be dumb luck!

I guess we'll find out... this is *the* single flakiest test in the editor suite, so it should be realtively obvious fairly quickly if it helps or not.

I'm also dubious, theoretically the `find` on the previous line should wait up to `max_wait_time` (20s?) for the input to be available...  but maybe the input is available but JS hasn't kicked in yet so we need the extra sleep?  I'll be honest, I didn't think much about this change! 😂 I just got pissed off enough to go and look at the test, saw a `sleep` and increased it! 

Maybe the 'proper' solution would be for the previous line to search for a css class that is only present once the autocomplete JS has done its thing...

Reply:
Ship it and see if it is anecdotally less flaky then, it won't add too long to the run time